parambokeh/__init__.py

```diff
@@ -114,7 +114,7 @@ def __call__(self, pname):
         if self.replace_underscores:
             pname = pname.replace('_',' ')
         if self.capitalize:
-            pname = pname.capitalize()
+            pname = pname[:1].upper() + pname[1:]
```

Comment:
Should it be capitalizing just the very first letter or the first letter of each word after splitting on spaces/underscores?

Reply:
I think only the very first lettter.  E.g. for "dynamic_network_address", I'd want "Dynamic network address", not "Dynamic Network Address".